src/site/content/en/blog/mouse-acceleration/index.md

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
Should we use `web.dev/disable-mouse-acceleration` instead? Would we ever want to talk about mouse acceleration more generally (not just disabling it, but other aspects of it)?